### ISSUE TITLE
Can not create nor modify media items in back office of Umbraco 8.10.0  

### ISSUE THREAD

**Issue by Jancornillie** [umbraco/Umbraco-CMS#9600]
When I want to modify (title or image) of media item the save button does nothing 

## Umbraco version

Umbraco 8.10.0

Reproduction
------------

Whether you:

- Go to Media section and try to modify title or image of existing image item.
- Or create a new media item, give it a title and upload an image.

In both cases clicking on the save button doesn't save the media item, without any warning or error message. Navigating away will give the message "Are you sure you want to navigate away from this page? - you have unsaved changes".

### Bug summary

One can not save media items in the back office.

### Specifics

url: /umbraco/#/media/media/edit/<ID>
Umbraco 8.10.0
Tried on Edge, Firefox and Chrome

### Expected result

After clicking on save I would expect modification being saved.

### Actual result

Nothing happens, no error message, nothing changed on screen.

**Comment by jorgecostadept** [comment]
I have the exact same issue. After upgrading to 8.10.0 in Umbraco Cloud I was not able to restore content anymore in my local environment because it's giving me a schema mismatch error with the Folder media type. I then found out that it's not possible to create new media items, unless you drag and drop and image, for example. Then it is created.

**Comment by jmayntzhusen** [comment]
Fairly certain this was introduced with https://github.com/umbraco/Umbraco-CMS/pull/9019

Changing the button type to submit works for me when testing.

**Comment by Jancornillie** [comment]
@jmayntzhusen 
Thanks,
Just did the same (changed the button type) and tested this in several browsers without any problem.

**Comment by mkanyo** [comment]
If you press "Enter" it works but yes, the Save button is dead.

**Comment by nathanwoulfe** [comment]
Hi @Jancornillie @jmayntzhusen @mkanyo @jorgecostadept this one's on me - I merged the changes in #9019 and obviously didn't test it properly (thinking I probably tested the fixed case, but not the original behavior). 

I also missed the follow up conversation on that issue, which suggests that the save button should be submit type (ie reverting the change in #9019), rather than button. If the submit-on-enter is an issue, it probably needs to be resolved in a different way. That also means that the save/save+publish on content items should really be a submit rather than button type, but that should be addressed separately.

I've added a PR to revert the change - #9604
